server-common/src/main/java/org/apache/kafka/timeline/Snapshot.java

```diff
@@ -24,10 +24,10 @@
  * A snapshot of some timeline data structures.
  * <br>
  * The snapshot contains historical data for several timeline data structures.
- * We use an IdentityHashMap to store this data.  This way, we can easily drop all of
+ * We use an IdentityHashMap to store this data.  This way, we can easily drop all
  * the snapshot data.
  */
-class Snapshot {
+public class Snapshot {
```

Comment:
Class is now public, but only read methods are public, so no class from outside the package can tamper with the data in this data structure